eclipse-collections-forkjoin/pom.xml

```diff
@@ -63,7 +63,21 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
+
+            <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19</version>
+                <configuration>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                    <argLine>-XX:-OmitStackTraceInFastThrow -Xms1024m -Xmx2048m @{argLine}</argLine>
+                    <runOrder>random</runOrder>
```

Comment:
Here and in other similar places everything except `argLine` (`version`, `includes`, `runOrder`) seems to be redundant since already declared in parent POM.